docs/system/yocto.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Yocto Build System
+tags:
+  - HMX
+  - MX-4
+  - CT
+  - MX-V
+  - T30
+  - T30 FR
+  - C61
+  - Development
+---
+
+## Overview
+
+The operating system running on Host Mobility Hardware and most of the tools are built from source. The build system used for this is the [Yocto Build System](https://www.yoctoproject.org/). This combines software written by Host Mobility with free software from software and hardware developers all over the world.  
+
+## Building with Yocto
+
+* To build packages and images, a Linux system is needed.  The quickest way is to use our Docker build script:
+
+*Clone mobility-poky-platform*
+```bash
+git clone git@github.com:hostmobility/mobility-poky-platform.git && cd mobility-poky-platform
+```
+
+*Build image for specific machine*
+```bash
+DL_DIR=$HOME/YOCTO_DOWNLOADS # Store downloads for reuse
+scripts/bygg \
+  --repo-sync \
+  --delete-conf \
+  --manifest-file kirkstone.xml \
+  --machine imx8mp-var-dart-hmx1 \
+  --image console-hostmobility-image \
+```
+
+*See [Building with Docker](yocto-build-with-docker.md) for details.*
```

Comment:
Why a bullet list first?

Also, I suggest using either proper headings (in markdown format) or regular text followed by a colon, e.g.

"Clone mobility-poky-platform:"